src/frontend/komponenter/Oppgaver/FilterSkjema.tsx

```diff
@@ -187,6 +187,7 @@ const FilterSkjema: React.FunctionComponent<IFilterSkjemaProps> = ({ innloggetSa
                 <Knapp
                     onClick={() => {
                         hentOppgaver(
+                            oppgaveGjengCap,
```

Comment:
Why send it as a parameter if it is globally defined?

Reply:
We need to extend the default 150 cap in the future.